src/components/lists/FamilyEventList.tsx

```diff
@@ -0,0 +1,137 @@
+import {
+  AbsoluteCenter,
+  Box,
+  Divider,
+  Flex,
+  useToast,
+  Text,
+  Button,
+} from '@chakra-ui/react'
+import { FamilyEvent } from '../family/FamilyEvent'
+import { IError, IEvent, TFamily } from '@/interfaces'
+import { TChildEntity } from '../forms/FamilyForm'
+import { deleteEvent } from '@/api/Events'
+import { WarningIcon } from '@chakra-ui/icons'
+
+type TProps = {
+  familyEvents: string[]
+  canModify: (
+    orgName: string | null,
```

Comment:
Yeh - we are using the same type here too

Reply:
Changes implemented